docs/content/ruby/guides/data_filtering_next/index.md

```diff
@@ -1,104 +1,68 @@
 ---
-title: "Filter Data - Adapter API (Preview)"
+title: "Filter Data (Preview)"
 weight: 1
 showContentForAnyLanguage: true
-no_nav: true
 ---
 
-# Filter Data - Adapter API (Preview)
+# New Data Filtering for Ruby
```

Comment:
Can we call this something other than "New Data Filtering"? I changed the title to "Adapter API" previously. I think we need a clearer way to disambiguate the two other than new / old for users.